### ISSUE TITLE
Romana agent fails to start

### ISSUE THREAD

**Issue by juris** [romana/romana#177]
Trying to use Romana with CoreOS 1576.5.0 and Kubernetes 1.8.7. Used kops 1.8.0 for provisioning.

All agents in DaemonSet fail to start:
```
Cluster "kubernetes" set.
User "romana-agent" set.
Context "romana-agent@kubernetes" modified.
Switched to context "romana-agent@kubernetes".
cp: cannot create regular file '/host/usr/local/bin/romana': Read-only file system
Checking if etcd is running...ok.
member c043523bec2f247 is healthy: got healthy result from http://etcd-b.internal.test.k8s.local:4001
member 28c405d29f7d5af0 is healthy: got healthy result from http://etcd-c.internal.test.k8s.local:4001
member b937d010fe885e77 is healthy: got healthy result from http://etcd-a.internal.test.k8s.local:4001
cluster is healthy
Checking if romana daemon is running...ok.
Checking if romana networks are configured...ok. one network configured.
Checking if policies are initialized...ok
Chain ROMANA_FORWARD (1 references)
target     prot opt source               destination         
RETURN     all  --  100.96.0.0/11        anywhere            
ACCEPT     all  --  anywhere             anywhere            
Launching Romana Agent
Build Revision: v2.0.1-bcb9912
Build Time: 2017-12-22_09:35:16AM
2018-01-31T12:01:02Z INFO     : IPAM exists at /ipam/data: true
2018-01-31T12:01:02Z INFO     : Loading IPAM data from /romana/ipam/data
2018-01-31T12:01:02Z WARN     : Romana gateway already exists.
2018-01-31T12:01:02Z ERROR    : failed to get default link: invalid argument
```

Other components are running without any issues.

**Comment by chrismarino** [comment]
Hi @juris, docs should be better on this point. Romana's installer does not support CoreOS right now. Working on it, though. @cgilmour has the details.

**Comment by cgilmour** [comment]
Hi @juris, a fix for that error (`failed to get default link: invalid argument`) has been made and will ship in the next point release. I'll put an update here when that next release is available.

**Comment by juris** [comment]
Perfect. Thanks!

**Comment by cgilmour** [comment]
Hi @juris , this should be resolved in the v2.0.2 update to Romana.
We will coordinate with the kops team for incrementing the versions that are used during a kops install.

**Comment by juris** [comment]
This is great news! Thank you Caleb.
